### PR TITLE
Add AXE

### PR DESCRIPTION
**Pull request by charlesrocket**
Ticker symbol: AXE
Official token name: Axe
Official block explorer URL: http://207.246.65.114:3001
Is it an Altcoin or a token: Altcoin
Official project URL: https://github.com/AXErunners/axe/wiki

### CODE REVIEW

File: common/src/main/java/io/bisq/common/locale/CurrencyUtil.java [hunk 1]
```diff
@@ -88,8 +88,8 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         final List<CryptoCurrency> result = new ArrayList<>();
 
         if (DevEnv.DAO_TRADING_ACTIVATED)
-            result.add(new CryptoCurrency("BSQ", "Bisq Token"));
-
+            result.add(new CryptoCurrency("AXE", "Axe"));
```
Comment (jonathancross) [inline common/src/main/java/io/bisq/common/locale/CurrencyUtil.java]: This doesn't look right... please revert that change.
Instead, AXE should be added a few lines down, just above this:
> result.add(new CryptoCurrency("BCH", "Bitcoin Cash"));

File: gui/src/test/java/io/bisq/gui/util/validation/AltCoinAddressValidatorTest.java [hunk 2]
```diff
@@ -487,4 +487,19 @@ public void testONION() {
         assertFalse(validator.validate("1HQQgsvLTgN9xD9hNmAgAreakzVzQUSLSH#").isValid);
         assertFalse(validator.validate("").isValid);
     }
+
+    @Test
+    public void testAXE() {
+        AltCoinAddressValidator validator = new AltCoinAddressValidator();
+        validator.setCurrencyCode("ONION");
```
Comment (jonathancross) [inline gui/src/test/java/io/bisq/gui/util/validation/AltCoinAddressValidatorTest.java]: Wrong currency code.

File: gui/src/test/java/io/bisq/gui/util/validation/AltCoinAddressValidatorTest.java [hunk 3]
```diff
@@ -487,4 +487,19 @@ public void testONION() {
...
+
+        assertTrue(validator.validate("XQZAWrECjc8jBN87TbqDvoDmcvWiftsmpq").isValid);
+        assertTrue(validator.validate("XYdqovfSUQvDEfmCzUCgc1FXtgxQsfCRWZ").isValid);
+        assertTrue(validator.validate("XZCz1HjLBQ7xAW94R4yhiQB6r6RsAuzRKx").isValid);
+
+        assertFalse(validator.validate("XQZAWrECjc8jBN87TbqDvoDmcvWiftsmpqq").isValid);
```
Comment (jonathancross) [inline gui/src/test/java/io/bisq/gui/util/validation/AltCoinAddressValidatorTest.java]: Please indent correctly.

**Comment by jonathancross** [comment]
Does not seem that this PR was tested. :(

File: common/src/main/java/io/bisq/common/locale/CurrencyUtil.java (hunk 1 above)
Comment (charlesrocket) [reply common/src/main/java/io/bisq/common/locale/CurrencyUtil.java]: got it

**Comment by ManfredKarrer** [comment]
Thanks @jonathancross for the review. That PR came with so sloppy issues that I am not willing to spend time on testing and merging. I also don't see the address validation code. Hire a pro Java dev and avoid wasting our time please.

**Comment by charlesrocket** [comment]
@ManfredKarrer wow theres a lot of aggression here) having troubles building for tests on osx 10.13 and some documentation is missing as well (https://github.com/bisq-network/exchange/wiki/How-to-prepare-your-Bisq-source-code-for-regtest-mode) we would be grateful if you could keep this PR open to avoid duplicates while we working on resolution. thank you so much

**Pull request closed without merging**